pygenometracks/tracks/HiCMatrixTrack.py

```diff
@@ -196,13 +196,19 @@ def plot(self, ax, chrom_region, region_start, region_end):
 
             elif self.properties['transform'] == '-log':
                 mask = matrix == 0
-                matrix[mask] = matrix[mask is False].min()
-                matrix = -1 * np.log(matrix)
+                try:
+                    matrix[mask] = matrix[mask == False].min()
+                    matrix = -1 * np.log(matrix)
+                except Exception:
```

Comment:
@joachimwolff should we catch a more specific exception here?

Reply:
Are you sure that the exception only happens if all values are cero? In that case you can simply check for `np.all(matrix == 0) `

Reply:
Can also be the case if the matrix contains no data.